core-bundle/contao/templates/twig/web_debug_toolbar/contao_collector.html.twig

```diff
@@ -115,6 +115,66 @@
         </tbody>
     </table>
 
+    {% if collector.backendSearchChecks|default %}
+        <h2>Back End Search</h2>
+        {% if collector.backendSearchChecks.available %}
+            <p>✅ The back end search is currently available.</p>
+        {% else %}
+            <p>🚨 The back end search is currently not available.</p>
+        {% endif %}
+        <table>
+            <thead>
+            <tr>
+                <th colspan="2">
+                    Requirements
+                </th>
+            </tr>
+            </thead>
+            <tbody>
+                <tr>
+                    <td>{% if collector.backendSearchChecks.sqlite_supported %}✅{% else %}🚨{% endif %}
+                    <td>
+                        {% if collector.backendSearchChecks.sqlite_supported %}
+                            The following PHP extensions are enabled: <code>{{ collector.backendSearchChecks.sqlite_supported|join('</code>, <code>') }}</code>.
+                        {% else %}
+                            One of the following PHP extensions need to be enabled: <code>pdo_sqlite</code>, <code>sqlite3</code>.
+                        {% endif %}
+                    </td>
+                </tr>
+                <tr>
+                    <td>{% if collector.backendSearchChecks.supervisor_supported %}✅{% else %}{% if collector.backendSearchChecks.cli_workers_running %}⚠️{% else %}🚨{% endif %}{% endif %}
```

Comment:
You don't need the supervisor. You can run the workers yourself. If `$webWorker->hasCliWorkersRunning()` returns true the workers are running, No matter whether they were triggered by our cronjob based supervisor or by a proper own process. So this message is only relevant if you also configured `contao.messenger.workers`. We should check for that on top, imho.

Reply:
Yes, the messages will reflect that. If the `SuperVisor` is supported, it does not matter how you process the queue (the next section covers that). If the `SuperVisor` is _not_ supported, it will be designated a warning, if the CLI workers are running anyway (i.e. you are not relying on the `SuperVisor`) and otherwise as an error.

Reply:
Mhm, it's just also something that's not necessarily only related to the backend search either. But it kinda makes sense here anyway.